pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -262,6 +265,13 @@ func getRejectOFPorts(rejectType RejectType, sIface, dIface *interfacestore.Inte
 		inPort = uint32(sIface.OFPort)
 		outPort = uint32(dIface.OFPort)
 	case RejectServiceLocal:
+		fallthrough
+	case RejectServiceLocalToRemote:
+		// For RejectServiceLocal and RejectServiceLocalToRemote, we set inPort as the
+		// OFPort of the srcPod to simulate its rejection. And we don't set outPort, since
+		// it's Service traffic loadBalanced by AntreaProxy. The reject response packet
+		// need to be UnDNATed by the pipeline, instead of directly sending it out
+		// through outPort.
 		inPort = uint32(sIface.OFPort)
```

Comment:
Not related to this PR, but I only see you add a few new reject type, so I am wondering where is the flow or rule to use this inPort to reject the traffic, could you point out? thanks.

Reply:
inPort is where this generated reject packet is injected into OVS. Not sure if I understand your question. Does this answer your question?